addons/sale_stock/tests/test_anglosaxon_account.py

```diff
@@ -9,6 +9,7 @@ class TestAngloSaxonAccounting(AccountTestInvoicingCommon):
     def setUpClass(cls):
         super().setUpClass()
         (cls.company_data['company'] + cls.company_data_2['company']).write({'anglo_saxon_accounting': True})
+        # fifo and real_time will be only set for company_A as product.category is company dependant
```

Comment:
I decided to add the com here instead so it's less weird than "trailing" comments